lib/Zonemaster/Backend/Config.pm

```diff
@@ -370,134 +348,67 @@ sub ListLanguageTags {
     return @langtags;
 }
 
-sub PollingInterval {
-    my ($self) = @_;
-
-    return $self->DB_polling_interval;
-}
-
-
-=head2 MaxZonemasterExecutionTime
-
-=head3 INPUT
 
-'max_zonemaster_execution_time' from [ZONEMASTER] section in ini file. See
-L<https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#max_zonemaster_execution_time>.
+=head2 DB_engine
 
-=head3 RETURNS
+Get the value of L<DB.engine|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#polling_interval>.
 
-Integer (number of seconds).
+Returns one of C<"SQLite">, C<"PostgreSQL"> or C<"MySQL">.
 
-=cut
 
-sub MaxZonemasterExecutionTime {
-    my ($self) = @_;
+=head2 DB_polling_interval
 
-    return $self->ZONEMASTER_max_zonemaster_execution_time;
-}
+Get the value of L<DB.polling_interval|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#polling_interval>.
 
 
-=head2 NumberOfProcessesForFrontendTesting
+=head2 ZONEMASTER_max_zonemaster_execution_time
 
-=head3 INPUT
+Get the value of L<ZONEMASTER.max_zonemaster_execution_time|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#max_zonemaster_execution_time>.
 
-'number_of_processes_for_frontend_testing' from [ZONEMASTER] section in ini file. See
-L<https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#number_of_processes_for_frontend_testing>.
+Returns an integer (number of seconds).
```

Comment:
Why specifying what kind of value is returned here and not for all the other one ?

Reply:
I'll remove it for consistency.